.gitignore

```diff
@@ -8,7 +8,6 @@ __pycache__
 *.egg-info*
 .python-version
 logs
-.vscode/
```

Comment:
Please leave this exclusion in place, vs code puts files in here for individual workspace configuration

Reply:
I can't do that and checkin a file for vscode's autoformatting rules.  We had decided that for spotless changes - and spotless - to stay in our processes, that we'd sync changes w/ IDEs so that devs wouldn't have to fight the formatters.  Is there another solution?

I would think that users would want to manage their own git ignores locally. Should a repo that doesn't use vscode for the solution to build need to ignore those directories globally (same argument for intelliJ and eclipse)?

Reply:
Git can still index files that are ignored, I've just moved all my settings from my local workspace to the machine wide configuration, I'll figure this out if it creates friction.